code/modules/mapping/space_management/space_reservation.dm

```diff
@@ -0,0 +1,70 @@
+
+//Yes, they can only be square.

```

Comment:
*Square* square, or just rectangular? Important difference.

Reply:
Appears to be rectangle, given proc `Reserve(width, height, zlevel)`.